client/lib/post-types-list/list.js

```diff
@@ -73,7 +75,7 @@ PostTypesList.prototype.fetch = function( siteId ) {
  * @return {array} services
  **/
 PostTypesList.prototype.parse = function( data ) {
-	return data.post_types;
+	return values( omit( data, '_headers' ) );
```

Comment:
I wonder if we should omit `_headers` field directly from the wpcom layer; either wpcom.js or another one.